app.js

```diff
@@ -30,14 +33,24 @@ app.use((err, req, res, next) => {
 	}
 })
 
+app.use((err, req, res, next) => {
```

Comment:
You can have multiple if statements in one app.use - usually a good idea to keep PSQL errors together, but no problem leaving them separate.